### PR TITLE
Add cleaning data for some mods by Scythe Bearer

### PR DESCRIPTION
**Pull request by BellCubeDev**
NOTE: These are the changes of Scythe Bearer, not me. I'm simply committing them. I take zero credit for these.

Added the plugins:
* [`__Fortune_Finder_Leveling.esp`](https://www.afkmods.com/index.php?/files/file/2410-fortune-finder-leveling/)
* [`__Wheres_the_rum.esp`](https://www.afkmods.com/index.php?/files/file/2420-wheres-the-rum/)
* [`__Working_Kitchen_Appliances.esp`](https://www.afkmods.com/index.php?/files/file/2414-working-kitchen-appliances/)
* [`_Complete_Raider_Power_Armor.esp`](https://www.afkmods.com/index.php?/files/file/2415-re-spawning-raiders-in-full-power-armor/)
* [`_Vault 88 Atruim Light.esp`](https://www.afkmods.com/index.php?/files/file/2415-re-spawning-raiders-in-full-power-armor/)

(I made slight stylistic changes to the YAML. [Original](https://www.dropbox.com/s/nex1pca2rv6ek9h/New%20Metadata.txt?dl=0))

Oh, and I've no clue where the plugins should go, so I just put them at the bottom. Could someone point me to the right location?

### CODE REVIEW

**Comment by Scythe-Bearer** [comment]
Thank you.  

The plugins can go pretty much anywhere.  They are standalone, with no dependencies.

File: masterlist.yaml [hunk 1]
```diff
@@ -6410,3 +6410,44 @@ plugins:
...
         util: '[FO4Edit v4.0.4b](https://www.nexusmods.com/fallout4/mods/2737)'
         itm: 4
+
+  - name: '__Fortune_Finder_Leveling.esp'
+    clean:
+      - crc: 0x3336af55
+        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+    url:
+      - 'https://www.afkmods.com/index.php?/files/file/2410-fortune-finder-leveling/'
+      - 'https://bethesda.net/en/mods/fallout4/mod-detail/4241028'
```
Comment (sibir-ine) [inline masterlist.yaml]: For consistency, the `url` key should be sorted above the `clean` key, a trailing slash should be added to the Bethesda.net Mods URL, & the cleaning data checksum should be capitalized. Also, the locations would benefit from names as they can be used in hyperlinks when displaying them:

```suggestion
    url:
      - link: 'https://www.afkmods.com/index.php?/files/file/2410-fortune-finder-leveling/'
        name: 'Fortune Finder Leveling (AFK Mods)'
      - link: 'https://bethesda.net/en/mods/fallout4/mod-detail/4241028/'
        name: 'Fortune Finder Leveling (Bethesda.net Mods)'
    clean:
      - crc: 0x3336AF55
        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -6410,3 +6410,44 @@ plugins:
...
+    
+  - name: '__Wheres_the_rum.esp'
+    clean:
+      - crc: 0xd05eea03
+        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+    url:
+      - 'https://www.afkmods.com/index.php?/files/file/2420-wheres-the-rum/'
+      - 'https://bethesda.net/en/mods/fallout4/mod-detail/4242093'
```
Comment (sibir-ine) [inline masterlist.yaml]: Same as above:

```suggestion
    url:
      - link: 'https://www.afkmods.com/index.php?/files/file/2420-wheres-the-rum/'
        name: 'Where''s The Rum (AFK Mods)'
      - link: 'https://bethesda.net/en/mods/fallout4/mod-detail/4242093/'
        name: 'Where''s The Rum (Bethesda.net Mods)'
    clean:
      - crc: 0xD05EEA03
        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
```

File: masterlist.yaml [hunk 3]
```diff
@@ -6410,3 +6410,44 @@ plugins:
...
+      
+  - name: '__Working_Kitchen_Appliances.esp'
+    clean:
+      - crc: 0x693933c3
+        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+    url:
+      - 'https://www.afkmods.com/index.php?/files/file/2414-working-kitchen-appliances/'
+      - 'https://bethesda.net/en/mods/fallout4/mod-detail/4241177'
```
Comment (sibir-ine) [inline masterlist.yaml]: Same as above:

```suggestion
    url:
      - link: 'https://www.afkmods.com/index.php?/files/file/2414-working-kitchen-appliances/'
        name: 'Working Kitchen Appliances (AFK Mods)'
      - link: 'https://bethesda.net/en/mods/fallout4/mod-detail/4241177/'
        name: 'Working Kitchen Appliances (Bethesda.net Mods)'
    clean:
      - crc: 0x693933C3
        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
```

File: masterlist.yaml [hunk 4]
```diff
@@ -6410,3 +6410,44 @@ plugins:
...
+      
+  - name: '_Complete_Raider_Power_Armor.esp'
+    clean:
+      - crc: 0x17ed1071
+        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+    url:
+      - 'https://www.afkmods.com/index.php?/files/file/2415-re-spawning-raiders-in-full-power-armor/'
+      - 'https://bethesda.net/en/mods/fallout4/mod-detail/4241257'
```
Comment (sibir-ine) [inline masterlist.yaml]: Same as above:

```suggestion
    url:
      - link: 'https://www.afkmods.com/index.php?/files/file/2415-re-spawning-raiders-in-full-power-armor/'
        name: 'Re-Spawning Raiders in Full Power Armor (AFK Mods)'
      - link: 'https://bethesda.net/en/mods/fallout4/mod-detail/4241257/'
        name: 'Re-Spawning Raiders in Full Power Armor (Bethesda.net Mods)'
    clean:
      - crc: 0x17ED1071
        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
```

File: masterlist.yaml [hunk 5]
```diff
@@ -6410,3 +6410,44 @@ plugins:
...
+    
+  - name: '_Vault 88 Atruim Light.esp'
+    clean:
+      - crc: 0x311e798e
+        util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+    url:
+      - 'https://www.afkmods.com/index.php?/files/file/2415-re-spawning-raiders-in-full-power-armor/'
+      - 'https://bethesda.net/en/mods/fallout4/mod-detail/4241257'
```
Comment (sibir-ine) [inline masterlist.yaml]: The same above applies. However, the URLs seem to have been copied & pasted from `_Complete_Raider_Power_Armor.esp`. @Scythe-Bearer, what are the locations of this mod?

**Review by sibir-ine — CHANGES_REQUESTED** [review]
If you want to move the mods to appropriate sections, you could add a `Gameplay - Economy & Item Balance` section for `__Fortune_Finder_Leveling.esp` & `__Wheres_the_rum.esp`. `__Working_Kitchen_Appliances.esp` could be placed in `Gameplay - Crafting, Harvesting & Scrapping`, & `_Complete_Raider_Power_Armor.esp` could be placed in `Gameplay - NPC Balance & Distribution`. I can't tell where `_Vault 88 Atruim Light.esp` could go without knowing it's purpose.

**Comment by Scythe-Bearer** [comment]
I have read this missive twice now, and suffice it to say that between my
inability to comprehend the documentation and my lack of understanding of
even the simplest concepts in this note, I am completely out of my depth.
So let us forget this attempt at getting my mods into the Loot Masterlist
until I get smarter, gain a grasp of the basic vocabulary, and can
effectively hold discourse on this subject.

I do thank you for your work and your attempts to help me, It is greatly
appreciated.

**Comment by sibir-ine** [comment]
Sorry for being unclear, most of that was directed at @BellCubeDev. I just pinged you because we need to know the link(s) for `_Vault 88 Atruim Light.esp` since you appear to have accidentally copied & pasted the links for `_Complete_Raider_Power_Armor.esp` in its locations.

**Comment by Scythe-Bearer** [comment]
The situation for me is unchanged.  In fact, the mod _Vault 88 Atruim
Light.esp hasn't even been published yet, so it is there because I screwed
the pooch.  It should be obvious to even the most casual observer that I do
not know what I am doing here, and I do not feel comfortable undertaking
this effort in such blind ignorance.

So , again, thank you for your efforts, but let's just pretend this never
happened.  I will try again later, when I am more educated.

**Comment by sibir-ine** [comment]
No worries, it can just be removed from the PR if it isn't publicly available yet. The other mods can be added when BellCube addresses my review, you don't need to worry about anything else.

File: masterlist.yaml [hunk 6]
```diff
@@ -6410,3 +6410,50 @@ plugins:
...
+
```
Comment (BellCubeDev) [inline masterlist.yaml]: In an effort to make this easier for the maintainers, it's a good 2 clicks for you guys to remove that section, if it is indeed what we're doing.
```suggestion
```

**Review by sibir-ine — APPROVED** [review]
Thanks!

**Comment by sibir-ine** [comment]
@Scythe-Bearer, the cleaning data for the four mods other than `_Vault 88 Atruim Light.esp` have been added to the masterlist. Thanks for submitting them, & sorry for the confusion.

**Pull request merged**